v0/destinations/yahoo_dsp/transform.js

```diff
@@ -0,0 +1,201 @@
+const { BASE_ENDPOINT, ENDPOINTS } = require("./config");
+const {
+  defaultRequestConfig,
+  CustomError,
+  defaultPutRequestConfig,
+  removeUndefinedAndNullValues,
+  getSuccessRespEvents,
+  getErrorRespEvents
+} = require("../../util");
+
+const {
+  getAccessToken,
+  populateIncludes,
+  populateExcludes,
+  createPayload
+} = require("./util");
+
+/**
+ * This function is used for building the final response to be returned.
+ * @param {*} message
+ * @param {*} destination
+ * @returns
+ */
+
+const responseBuilder = async (message, destination) => {
+  const { listData } = message.properties;
+  const {
+    accountId,
+    audienceId,
+    audienceType,
+    seedListType
+  } = destination.Config;
+
+  let outputPayload = {};
+  // The only supported property for now is "add"
+  const key = "add";
+
+  const domains = [];
+  const categoryIds = [];
+  const listDataKey = listData[key];
```

Comment:
This variable contains the user identifier please choose a name which is meaningful